src/model/ChillerElectricEIR.hpp

```diff
@@ -147,6 +149,18 @@ namespace model {
 
     std::string endUseSubcategory() const;
 
+    std::string condenserFlowControl() const;
+
+    boost::optional<Curve> condenserLoopFlowRateFractionFunctionofLoopPartLoadRatioCurve() const;
+
+    boost::optional<Schedule> temperatureDifferenceAcrossCondenserSchedule() const;
+
+    boost::optional<double> condenserMinimumFlowFraction() const;
```

Comment:
@joseph-robertson I see you made that one required, the getter should be returning a double.